nodejs/week1/meal_sharing_website/src/backend/routes/reservation.js

```diff
@@ -0,0 +1,20 @@
+//Respond with the json for a random reservation
+
+const express = require("express");
+const router = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.